app/javascript/components/patient/laboratory/FirstPositiveLab.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button } from 'react-bootstrap';
+import moment from 'moment';
+
+import LaboratoryModal from './LaboratoryModal';
+
+class FirstPositiveLab extends React.Component {
```

Comment:
for naming consistency with other lab components, can we call this `FirstPositiveLaboratory`?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/0b00f9efd83684aa21526ddee93b433c20c8fb79#diff-8147b18fdde2f3d544f4ab613568fc11952197eb2907e5d2a55ae37756aea1a2R8